Vagrantfile

```diff
@@ -26,28 +26,36 @@ end
 
 Vagrant.configure(2) do |config|
 
-    config.vm.box = "debian/jessie64"
+    # Prefer docker over virtualbox since it is listed first
+    # meaning that docker should run if no '--provider' is supplied
+    # when calling vagrant up
+    config.vm.provider "docker" do |d, configOverride|
+        d.build_dir = "."
+        d.pull = true
+        d.has_ssh = true
+
+        # Overrides for 'config' unique for docker
+        configOverride.ssh.username = "vagrant"
+        configOverride.ssh.password = "vagrant"
+    end
 
-    config.vm.provider "virtualbox" do |vb|
+    config.vm.provider "virtualbox" do |vb, configOverride|
```

Comment:
@rpannek .. which will load up a virtual machine.. I guess that will work, not sure if it is better than using VBox directly though.